.golangci.yml

```diff
@@ -67,8 +68,16 @@ linters:
             - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
             - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
             - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
+    importas:
+      alias:
+        # we can add more layers here (containerlab$1$2 etc.) as we tidy/get more consistent
```

Comment:
obv up to the project how crazy we go here, for *me* personally... pedantic though it may be (and maybe annoyingly verbose) I think being really explicit w/ import aliases and having things be fully qualified is really nice for readability